### ISSUE TITLE
Is this Repo Still Relevant

### ISSUE THREAD

**Issue by sunildhiman90** [epm-dev-priporov/ktor-cloud#1]
Hi, 

Just curious to know, if this repo is still relevant for creating micro services using Ktor server? I am asking this as i cannot see any commit since last 4 years almost.

Thinking of using it for micrservices in ktor just like we use eureka service discovery and registration in spring boot something like this: https://spring.io/guides/gs/service-registration-and-discovery.

Thanks

**Comment by epm-dev-priporov** [comment]
hello @sunildhiman90
I really didn't update it pretty long time, because there was no any fidback. If you need I could update this to required ktor/jdk/kotlin versions

**Comment by sunildhiman90** [comment]
Hey yes please., i would like to use. 

 
But i have one question can we use it like the example link i posted. I means for service discovery and registeration?? 

May be if possible you can create one sample of ktor microservices connected using this same as we do in spring boot. So that it will be easier to use this.

Thanks.

**Comment by epm-dev-priporov** [comment]
@sunildhiman90 do you mean example where ktor works as eureka server? I am afraid I didn't implemen it. only eureka client

**Comment by sunildhiman90** [comment]
Yes, i was thinking of something like creating  microservices using erureka clients and service discovery as well.
